lib/file.js

```diff
@@ -250,6 +250,10 @@ FileCoverage.prototype.merge = function (other) {
         var i,
             retArray = that.data.b[k],
             secondArray = other.b[k];
+        if (!retArray) {
+            that.data.b[k] = secondArray;
```

Comment:
I think it might be worth assigning `that.data.b[k]` to a copy of secondArray, rather than assigning secondArray directly; seems less likely that we have two things modify `secondArray` accidentally.